### PR TITLE
Change visuals and add story

### PR DESCRIPTION
**Pull request by Dimi20cen**
- I changed the visuals of index's and mercury's pages
- I added a small story for mercury

If the changes are welcome, I'll continue with the rest of the planets.
@chimosky

### CODE REVIEW

**Comment by chimosky** [comment]
> If the changes are welcome, I'll continue with the rest of the planets.

Yes the changes are welcome, please continue.

File: data/mercury.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  }
+  
+  ul {
+    padding-left: 20px;
+  }
+  
+  ul li {
+    line-height: 1.6;
+  }
+
```
Comment (chimosky) [inline data/mercury.css]: Could you remove the No new line at EOF?

**Comment by chimosky** [comment]
Reviewed 9bef8e2, would be really helpful if your commit messages aren't just "Update *.html".

**Comment by Dimi20cen** [comment]
Would a commit message along those lines be preferable e.g. for earth.html
- Add <html> element
- Add <meta> element for responsive web design
- Add <meta> element "Roboto" font
- Change link to css file
- Remove <center> element around first img
- Change location of images on page
- Add inline css for images to configure their placement, their appearance and their size
- Add short fun story about earth
- Add inline css for the "Significant Dates:" and "Quick Earth Facts" to align them to the left of the page
- Change element for the credits section to <footer>
- Replace h3 with h2
- Remove spaces between <p> elements

**Comment by chimosky** [comment]
> Would a commit message along those lines be preferable e.g. for earth.html
> 
>     * Add  element
> 
>     * Add  element for responsive web design
> 
>     * Add  element "Roboto" font
> 
>     * Change link to css file
> 
>     * Remove  element around first img
> 
>     * Change location of images on page
> 
>     * Add inline css for images to configure their placement, their appearance and their size
> 
>     * Add short fun story about earth
> 
>     * Add inline css for the "Significant Dates:" and "Quick Earth Facts" to align them to the left of the page
> 
>     * Change element for the credits section to
> 
>     * Replace h3 with h2
> 
>     * Remove spaces between  elements

This seems better, you can lose the tag identifiers as they're not necessary.

**Comment by Dimi20cen** [comment]
What do you mean by "lose the tag identifiers"?

**Comment by chimosky** [comment]
> What do you mean by "lose the tag identifiers"?

In your response above you used `Added <html> element` and in markdown using <> which identifies html tags hides the tag you're trying to include in your comment and they're not necessary for your commit message.

**Comment by Dimi20cen** [comment]
I made one big commit instead of smaller one. I did that because 
- the .css file is now used universally from all .html files
- the changes to each page are very similar to each other

**Comment by chimosky** [comment]
> I made one big commit instead of smaller one. I did that because
> 
>     * the .css file is now used universally from all .html files
> 
>     * the changes to each page are very similar to each other

I had no issues with the size of the commit diff, my comment was about your commit messages.

**Comment by Dimi20cen** [comment]
Just to clarify is the current commit good?

**Comment by chimosky** [comment]
The latest commit that details your changes is good.

**Comment by Dimi20cen** [comment]
Hey, I'm looking at the maintainer's checklist and I'm not sure of what I can do to help in the release of this and the others activities. Could you help me with that?

**Comment by quozl** [comment]
Thanks for asking.  I've had a quick look at this pull request and have the following comments;

* [ ] the commit messages don't explain what is done and why it was done, which has been mentioned already,
* [ ] the source attributions to NASA have been removed,
* [ ] several new astronomical images have been added but without copyright, license, or attribution,
* [ ] fonts have been added that require network access, yet the activity was designed to be used when network was not available.

(Disclosure; in another role, I work for the Australian National University, Research School of Astronomy and Astrophysics, at the Anglo-Australian Telescope as a telescope operator and software engineer.  My comments here are not related to that role, and I'm not representing either the ANU or the AAT in my review of this pull request.  I'm answering as a member of Sugar Labs.)

**Comment by Dimi20cen** [comment]
Thank you for your help, could you help me a bit more?

> The commit messages don't explain what is done

Could you clarify what you mean by this? Are you suggesting that the commit messages should describe the changes in a higher-level context? E.g, should they state that the updates focus on adding narrative elements to each page and aim to improve the overall user experience?

> The commit messages don't explain why the changes was done

Should I add the reasoning in a new commit or should I edit the commit that I have already made?

> the source attributions to NASA have been removed

I started with changing the mars page whose source page didn't work (the actual link has an "in-depth" part, however in the referenced link it's written "indepth", which may have caused the issue in the mars' case) I made a wrong assumption on the rest. WIll fix.

>several new astronomical images have been added but without copyright, license, or attribution

The images I used are from Unsplash, which operates under a license that does not require permission or attribution. However, I understand the importance of proper attribution. Could you advise on whether I should add these attributions in the README file or directly at the bottom of each respective page or should I attribute in a different way?

> fonts have been added that require network access, yet the activity was designed to be used when network was not available.

I didn't even think about that, thanks for the insight, will fix.

**Comment by quozl** [comment]
> > The commit messages don't explain what is done [ and why it was done ]
> 
> Could you clarify what you mean by this? Are you suggesting that the commit messages should describe the changes in a higher-level context? E.g, should they state that the updates focus on adding narrative elements to each page and aim to improve the overall user experience?

Here's what our guide to contributing says;

https://github.com/sugarlabs/sugar-docs/blob/master/src/contributing.md#making-commits

> 1. start with a one line summary of the change;
> 2. leave a blank line after the summary;
> 3. explain the problem that is solved, unless the summary makes it obvious;
> 4. when the problem was introduced by a previous commit, mention the hash;
> 5. when the problem is in an issue or ticket, add "Fixes #1234";
> 6. avoid mentioning GitHub or other pull-requests, as these are not kept in git;
> 7. avoid mentioning any contest tasks or mentors; use pull-request comments instead; and
> 8. use imperative mood, like "add foo", or "port to bar"; (if English is not your first language, see [imperative mood](https://en.wikipedia.org/wiki/Imperative_mood), [git documentation](https://git.kernel.org/pub/scm/git/git.git/tree/Documentation/SubmittingPatches#n133) and [blog post by Dan Clarke](https://www.danclarke.com/git-tense)).

There are other resources available outside Sugar Labs that explain how to write commit messages for review, but the above is what we have agreed upon.

Further on, the guide for reviewers says in part;

> ### Goals
> * maintain a useful git commit history for use by git bisect, and developers who read it,
> ### Checklist
> * [ ] does the commit message explain the summary, problem, and solution, so that it can be used in future analysis, see also [making commits](#making-commits) (if a reviewer can fix it by squash or manual rebase, do so),
>
> * [ ] does the commit message reference any issue, bugs.sugarlabs.org ticket number, or downstream ticket numbers, (if a reviewer can fix it by squash or manual rebase, do so),
>
> * [ ] are the number of commits excessive for future analysis, (a reviewer may squash or rebase if necessary),
>
> * [ ] is the changed code consistent in style with the existing code, see also [coding standards](desktop-activity.md#coding-standards), (on the other hand, expect flake8 changes to be in separate commits),

On the basis of the above, your commit messages do not pass review.

> > The commit messages don't explain why the changes was done
> 
> Should I add the reasoning in a new commit or should I edit the commit that I have already made?

I'd prefer to review commits in their own context; the changes and the commit message should be together in the same commit.  That's because I use GitHub view of each commit, and several other tools that view commits one by one.  Commits are what are kept in git, so commits have to be consistent within themselves, or there's very little point keeping the history of changes and we'd just ask you to release a new bundle file.

I suggest you edit the commit messages for each commit.  Or if you realise that the way you have made the existing commits doesn't properly support review, you might create a new chain of commits entirely.  Either way, the commit hashes will change, and review will start again.

> > several new astronomical images have been added but without copyright, license, or attribution
> 
> The images I used are from Unsplash, which operates under a license that does not require permission or attribution. However, I understand the importance of proper attribution. Could you advise on whether I should add these attributions in the README file or directly at the bottom of each respective page or should I attribute in a different way?

Thanks.  Up to you how to attribute, as long as it is in a commit and not in a GitHub pull request comment.  Pull requests are not kept in git.

**Comment by Dimi20cen** [comment]
Again thank you for the help, I squased the previous commits into 3 to keep the commit history tidy. 
When I'm writing commit messages I feel a bit lost with how deep/detailed should the explaination of the solution be. I hope those commit messages are up to part.

**Comment by quozl** [comment]
Thanks.  Reviewed.

* dc87044, the `<br>` tags seem unnecessary, and the last three lines are indented differently,
* e5c05ba,
  * the Roboto font is Apache 2.0 licensed, yet the license is not included, nor is the `activity.info` changed; are you sure you want to dual-license the activity, can't you just use the default or always present fonts?  Children take text in fastest when a single font is used across all content, because of neural network reuse.  What is your justification for the font change?  You say improved readability, but on what basis?  What font was used before your change?
  * the Roboto font continues to be referenced in `<head><link>` via the network, yet is included as a file,
* 6dff579,
  * the Roboto font continues to be referenced in `<head><link>` via the network, yet is included as a file,
  * "the images used on this website" should be "the images used in this activity",
  * the new narrative descriptions don't appear to be factual and could be misleading, depending on culture, and they are _not what NASA said_, so the attribution is inaccurate,
  * much of the data seems wrong; for example the Neptune year and number of moons, which is probably because you assumed that commented-out quick facts sections were real data, when in fact they were probably copy and paste without completion; see for instance how the radius 69,911km appears in _four_ planet files before your commits.

**Comment by Dimi20cen** [comment]
Thanks again,
- dc87044, yep `<br>` will be removed. The last three lines all use block-quotes. Also in the preview they indentation looks the same, what am I missing?
- e5c05ba,
  - Roboto seemed an easier to read font, much less detailed than the default. However, I tested mostly in a browser that defaulted to Liberation Serif, while in Sugar's environment I think the font used is DejaVu Sans. Considering this, Roboto makes no sense.
- 6dff579,
  - By "new narrative descriptions" do you mean the stories? Their purpose is to make children more engaged/invested with the solar-system so they are more inclined to learn about them. Regarding NASA attribution, should we make it explicit that these stories are original and not sourced from NASA?
  - You are correct about the 'quick facts' section. I had assumed their accuracy, but I'll replace them with verified information.
<br>
I'm thinking of editting the commits I have already made, to keep the history tidy. Will that make reviewing and future history inspecting easier or should I make a new commit(s) that contains the suggested changes?

**Comment by chimosky** [comment]
> By "new narrative descriptions" do you mean the stories? Their purpose is to make children more engaged/invested with the solar-system so they are more inclined to learn about them. Regarding NASA attribution, should we make it explicit that these stories are original and not sourced from NASA?

Please don't give attribution where not necessary, if something isn't from NASA then don't say it is.

> I'm thinking of editting the commits I have already made, to keep the history tidy. Will that make reviewing and future history inspecting easier or should I make a new commit(s) that contains the suggested changes?

No need to edit commits as it costs nothing to create new ones and commit history exists to show your thought process so make new commits with the suggested changes.

**Comment by Dimi20cen** [comment]
> Please don't give attribution where not necessary, if something isn't from NASA then don't say it is.

Hundred percent agree. In a practical sense, should I clarify at the bottom of the page that the stories are originally made and not sourced from NASA?

> No need to edit commits as it costs nothing to create new ones and commit history exists to show your thought process so make new commits with the suggested changes.

Got it, I was afraid that a lot of commits might make the commit history harder to understand. Also does a commit message with the title of "Resolve Reviews" would be a good idea? Thank you!

**Comment by chimosky** [comment]
> Hundred percent agree. In a practical sense, should I clarify at the bottom of the page that the stories are originally made and not sourced from NASA?

The commit messages tell us that the stories are originally made and the only time you'd want to mention NASA is if you got the story from them and if you didn't then there's no need to mention them.

> Got it, I was afraid that a lot of commits might make the commit history harder to understand. Also does a commit message with the title of "Resolve Reviews" would be a good idea? Thank you!

If the change is functional then you can explain what the change does with the title specifying it too, if the change is about code style then you can have that as your title and the body explaining that the change is for code consistency.

You also seem to write your message body right below your commit message title, it'll be nice to leave a new line before writing your message body, you should take a look at our [contributing guide](https://github.com/sugarlabs/sugar-docs/blob/master/src/contributing.md#making-commits).

**Comment by chimosky** [comment]
I forgot to add earlier that if the facts you add are from a certain source then attributing the facts to the source should be done.

**Comment by Dimi20cen** [comment]
Yep, facts are from nasa's website, except from the stories

**Comment by Dimi20cen** [comment]
Hey, can we merge?

**Comment by chimosky** [comment]
I see you updated quick facts in 43501ed, could you add the sources for the update in your commit message?

**Comment by Dimi20cen** [comment]
Done

**Comment by chimosky** [comment]
Reviewed, I'll wait for @quozl to review.

**Comment by Dimi20cen** [comment]
Thank you!

**Pull request merged**